lib/Runtime/Base/ScriptContext.cpp

```diff
@@ -1878,35 +1960,16 @@ namespace Js
                 *heap = nullptr;
             }
 
-            Var* localModuleFunctions = moduleMemoryPtr + wasmModule->funcOffset;
-
-            FrameDisplay * frameDisplay = RecyclerNewPlus(GetRecycler(), sizeof(void*), FrameDisplay, 1);
-            frameDisplay->SetItem(0, moduleMemoryPtr);
-
-            // TODO, refactor this function into smaller functions
-            for (uint i = 0; i < wasmModule->functions->Count(); ++i)
-            {
-                AsmJsScriptFunction * funcObj = javascriptLibrary->CreateAsmJsScriptFunction(functionArray[i]->body);
-                funcObj->GetDynamicType()->SetEntryPoint(AsmJsExternalEntryPoint);
-                funcObj->SetModuleMemory(moduleMemoryPtr);
-                FunctionEntryPointInfo * entypointInfo = (FunctionEntryPointInfo*)funcObj->GetEntryPointInfo();
-                entypointInfo->SetIsAsmJSFunction(true);
-                entypointInfo->address = AsmJsDefaultEntryThunk;
-                entypointInfo->SetModuleAddress((uintptr_t)moduleMemoryPtr);
-                funcObj->SetEnvironment(frameDisplay);
-                localModuleFunctions[i] = funcObj;
-            }
-
             for (uint32 iExport = 0; iExport < wasmModule->info->GetExportCount(); ++iExport)
             {
                 Wasm::WasmExport* funcExport = wasmModule->info->GetFunctionExport(iExport);
-                if (funcExport)
+                if (funcExport && funcExport->nameLength > 0)
```

Comment:
Would it make sense to emit an error if an function named "memory" was marked for export?